general-concepts/dependencies/text.xml

```diff
@@ -850,5 +850,309 @@ USE flags. This would then break building your ebuild.
+<c>PDEPEND</c> guarantees only that at least one is installed for runtime.
+You should not assume the same alternative or slot will be used for both.
+</p>
+
+</body>
+</subsection>
+
+<subsection>
+<title>The <c>:=</c> slot operator across classes</title>
+<body>
+
+<p>
+The <c>:=</c> operator is technically valid in all classes but is useful only
+when the matching package is installed at the time metadata are recorded
+(install from source or binary package creation). Practically, the atom using
+<c>:=</c> should be present in <c>RDEPEND</c> to express the rebuild
+relationship, and <c>DEPEND</c> must guarantee that a matching package is
+installed at the relevant time.
+</p>
+
+<codesample lang="ebuild">
+# Typical pattern — ensure the provider is present at build time and tie
+# rebuilds to its sub-slot changes
+DEPEND="dev-libs/foo:="
+RDEPEND="dev-libs/foo:="
+</codesample>
+
+<note>
+The atoms need not be textually identical as long as <c>DEPEND</c> guarantees
+that some package matching the <c>RDEPEND</c> <c>:=</c> atom is installed when
+metadata are recorded.
+</note>
+
+</body>
+</subsection>
+
+<subsection>
+<title>Blockers across dependency classes</title>
+<body>
+
+<p>
+Blockers are valid in all classes but their usefulness differs:
+</p>
+
+<ul>
+  <li>
+    Weak blockers (<c>!</c>) are primarily meaningful in <c>RDEPEND</c>, where
+    the uninstall of the blocked package may be delayed until after the new
+    package starts installing, allowing file collisions to be replaced.
+    Technically they are allowed in <c>DEPEND</c>, but they do not influence
+    the build environment and are not useful there on their own.
+  </li>
+  <li>
+    Strong blockers (<c>!!</c>) must be resolved before the dependency is
+    considered satisfied. They therefore make sense in both <c>DEPEND</c>
+    (before building) and <c>RDEPEND</c> (before installation).
+  </li>
+  <li>
+    In <c>PDEPEND</c>, weak blockers act effectively like those in
+    <c>RDEPEND</c>, while strong blockers behave more like weak ones because
+    satisfaction may be delayed post-install.
+  </li>
+  <li>
+    Remember the general caveat from above: weak blockers should be included
+    in <c>RDEPEND</c> rather than used purely in <c>DEPEND</c>.
+  </li>
+
+</ul>
+
+<codesample lang="ebuild">
+# Weak blocker — meaningful in RDEPEND
+RDEPEND="!app-misc/foo"
+
+# Strong blocker — enforced pre-build or pre-install
+DEPEND="!!sys-libs/bar"
+</codesample>
+
+</body>
+</subsection>
+
+<subsection>
+<title>Further reading</title>
+<body>
+
+<p>
+For extended discussion with examples, see the
+<uri link="https://blogs.gentoo.org/mgorny/2016/06/21/dependency-pitfalls-regarding-slots-slot-ops-and-any-of-deps/"
+>blog post</uri>
+and its
+<uri link="https://blogs.gentoo.org/mgorny/2016/06/22/dependency-classes-and-allowed-dependency-types/"
+>follow-up</uri>,
+and the relevant sections of PMS on any-of dependency specifications and slot
```

Comment:
Can you link to https://projects.gentoo.org/pms/8/pms.html#x1-760008.2.3 and https://projects.gentoo.org/pms/8/pms.html#x1-820008.3.3 while you're at it?